lib/winevt/subscribe.rb

```
@@ -2,10 +2,23 @@ module Winevt
   class EventLog
     class Subscribe
       alias_method :subscribe_raw, :subscribe
+      alias_method :initialize_raw, :initialize
+      attr_reader :query_error
+
+      def initialize
+        @query_error = nil
+        initialize_raw
+      end
 
       def subscribe(path, query, bookmark = nil)
         if bookmark.is_a?(Winevt::EventLog::Bookmark)
-          subscribe_raw(path, query, bookmark.render)
+          begin
+            subscribe_raw(path, query, bookmark.render)
+          rescue Winevt::EventLog::Query::Error => e
+            @query_error = e
+            # fallback to without bookmark version.
+            subscribe_raw(path, query)
```

I'm not sure the Windows API.
Is it OK to ignore the error?